### PR TITLE
Manual merge

### PR DESCRIPTION
**Pull request by franmomu**

### CODE REVIEW

File: src/Templating/MutableTemplateRegistryAwareInterface.php [hunk 1]
```diff
@@ -20,7 +20,7 @@ interface MutableTemplateRegistryAwareInterface
 {
     public function getTemplateRegistry(): MutableTemplateRegistryInterface;
 
-    public function setTemplateRegistry(MutableTemplateRegistryInterface $templateRegistry): void;
+    public function setTemplateRegistry(TemplateRegistryInterface $templateRegistry): void;
```
Comment (franmomu) [inline src/Templating/MutableTemplateRegistryAwareInterface.php:23]: Should we change `AbstractAdmin::setTemplateRegistry` in `3.x`? @VincentLanglet @wbloszyk I'm not familiar with all the changes that happened/are happening with the templates.

Comment (VincentLanglet) [reply src/Templating/MutableTemplateRegistryAwareInterface.php:23]: I would say that this change should not be made instead.
The getter expect to return `MutableTemplateRegistryInterface` so the setter should expect the same thing.

Which error did you try to solve ?

**Review by VincentLanglet — CHANGES_REQUESTED** [review]

File: src/Templating/MutableTemplateRegistryAwareInterface.php (hunk 1 above)
Comment (VincentLanglet) [reply src/Templating/MutableTemplateRegistryAwareInterface.php:23]: Oh seems like there is something to fix in 3.x
https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Templating/MutableTemplateRegistryAwareInterface.php#L21
and
https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Templating/MutableTemplateRegistryAwareInterface.php#L32
have different typehint.

Comment (VincentLanglet) [reply src/Templating/MutableTemplateRegistryAwareInterface.php:23]: Seems like I introduced this, don't know why.

I made https://github.com/sonata-project/SonataAdminBundle/pull/6606

**Review by VincentLanglet — APPROVED** [review]

**Pull request merged**